_episodes/04-drawing-bitwise.md

```diff
@@ -414,39 +371,34 @@ The resulting masked image should look like this:
 > >  * Python program to mask out everything but the wells 
 > >  * in a standardized scanned 96-well plate image.
 > > '''
-> > import cv2
 > > import numpy as np
+> > import skimage
+> > from skimage.viewer import ImageViewer
 > > import sys
 > > 
-> > # read in original image and open the centers file
-> > originalImage = cv2.imread(filename = sys.argv[1])
-> > centerFile = open(file = 'centers.txt')
+> > # read in original image
+> > image = skimage.io.imread(sys.argv[1])
 > > 
 > > # create the mask image
-> > mask = np.zeros(shape = originalImage.shape, dtype='uint8')
+> > mask = np.ones(shape = image.shape[0:2], dtype='bool')
 > > 
-> > # iterate through the centers file...
-> > for line in centerFile:
-> >     # ... getting the coordinates of each well...
-> >     tokens = line.split()
-> >     x = int(tokens[0])
-> >     y = int(tokens[1])
+> > # open and iterate through the centers file...
+> > with open("centers.txt", "r") as center_file:
```

Comment:
I couldn't test this excercise - `centers.txt` seems to be missing from this repo.

Reply:
@mmeysenburg - could you please advise on locating the `centers.txt` file? I vaguely remember this being a file that learners created during the workshop, but would be useful to have a copy in the repo to test the code and for Instructors checking solutions.

Reply:
Hey @mmeysenburg, in preparation of the pilot here in Heidelberg we would really like to use this exercise. For this, however, we'd need the `centers.txt` file. Could you please have another look?
Cheers